app/config/default.json

```diff
@@ -1,4 +1,5 @@
 {
+  "version": "1.7.0",
```

Comment:
What will happen if the config is moved out of the app/config folder? the version replace command wont be able to make that update